docs/source/blogs/release-0-3-0-blog/release-0-3-0-blog.rst

```diff
@@ -0,0 +1,192 @@
+
+.. code-block::
+
+  from pyspark.sql.functions import expr
+  function_normalized_variants_df = original_variants_df.withColumn( \
+    "normalizationResult", \
+    expr("normalize_variant(contigName, start, end, referenceAllele, alternateAlleles, '/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa')") \
+  )
+
+As discussed in the previous section, this SQL expression function, like any other in Glow, now has Python and Scala APIs as well. Therefore, the same can be done in Python as follows:
+
+.. code-block::
+
+  from glow.functions import normalize_variant
+  function_normalized_variants_df = original_variants_df.withColumn( \
+    "normalizationResult", \
+    normalize_variant( \
+      "contigName", \
+      "start", \
+      "end", \
+      "referenceAllele", \
+      "alternateAlleles", \
+      "/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa" \
+    ) \
+  )
+
+and in Scala as well, assuming ``original_variant_df`` is defined in Scala:
+
+.. code-block:: scala
+
+  import io.projectglow.functions.normalize_variant
+  import org.apache.spark.sql.functions.col
+  val function_normalized_variants_df = original_variants_df.withColumn(
+    "normalizationResult",
+    normalize_variant(
+      col("contigName"),
+      col("start"),
+      col("end"),
+      col("referenceAllele"),
+      col("alternateAlleles"),
+      "/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa"
+    )
+  )
+The result of any of the above commands will be the same as :numref:`fignormnorepdf`.
+
+
+A new transformer to split multiallelic variants
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This release also introduced a new DataFrame transformer, called ``split_multiallelics``, to split multiallelic variants into biallelic ones with a behavior similar to `vt decompose <https://genome.sph.umich.edu/wiki/Vt#Decompose>`_ with ``-s`` option. This behavior is significantly more powerful than the behavior of the previous version's splitter which behaved like GATK’s `LeftAlignAndTrimVariants <https://gatk.broadinstitute.org/hc/en-us/articles/360037225872-LeftAlignAndTrimVariants>`_ with ``--split-multi-allelics``. In particular, the array-type ``INFO`` fields and genotype fields with entries corresponding to each of alternate alleles and/or reference allelles are split "smart"ly (see ``-s`` option of `vt decompose <https://genome.sph.umich.edu/wiki/Vt#Decompose>`_) into biallelic rows. So are the genotype fields with entries corresponding to all possible genotype calls sorted in colex order, e.g., the ``GL``, ``PL``, and ``GP`` fields of the VCF format. Moreover, an ``OLD_MULTIALLELIC`` ``INFO`` field is added to the DataFrame to store the original multiallelic form of the biallelic variants resulting from splitting.
```

Comment:
Remove `significantly`
Replace `smartly` -> `intelligently`
I'm a little confused by the two sentences starting with `In particular`. Can you clean up the wording?